ingest/cli_parser.py

```diff
@@ -331,6 +331,63 @@ def create_parser():
         "--gene-file", help="Path to .genes.tsv file"
     )
 
+    parser_ingest_differential_expression = subparsers.add_parser(
+        "ingest_differential_expression",
+        help="Indicates author differential expression analysis processing",
+    )
+
+    parser_ingest_differential_expression.add_argument(
+        "--ingest-differential-expression",
+        required=True,
+        action="store_true",
+        help="Indicates that author differential expression analysis should be invoked",
+    )
+
+    parser_ingest_differential_expression.add_argument(
```

Comment:
It doesn't break anything, but I don't know that we need `--study-accession` for DE jobs.  Typically it's only needed for metadata jobs since we need that value in BigQuery.

Reply:
You're right it's not needed, but having study accession available at a glance in logs, Sentry, etc. substantially improves debuggability.